lib/form_data.js

```diff
@@ -235,7 +235,12 @@ FormData.prototype._getContentDisposition = function(value, options) {
   }
 
   if (filename) {
-    contentDisposition = 'filename="' + filename + '"';
+    // Escaping to prevent CRLF Injection / tampering with invalid fields.
+    // https://html.spec.whatwg.org/#multipart-form-data
+    var escapedFilename = filename.replaceAll('"', '%22')
```

Comment:
unfortunately replaceAll isn't available in all engines we support. what about this?
```suggestion
    var escapedFilename = filename.replace(/"/g, '%22')
```